site/documentation/content/user-guide/mqtt-adapter.md

```diff
@@ -43,6 +43,10 @@ The examples below refer to devices `4711` and `gw-1` of tenant `DEFAULT_TENANT`
 
 After verifying the credentials, the number of existing connections is checked against the configured [resource-limits] ({{< ref "/concepts/resource-limits.md" >}}) by the MQTT adapter.  If the limit is exceeded then a return code `0x05` indicating `Connection Refused: not authorised` is sent back.
 
+## Connection Duration Limits
+
+Before accepting any connection requests from the devices, the MQTT adapter verifies that the configured [connection duration limit] ({{< relref "/concepts/resource-limits.md#connection-duration-limit" >}}) is not exceeded. If the limit has been already reached, then a return code `0x05` indicating `Connection Refused: not authorised` is sent back.
```

Comment:
this also needs to be added to the Kura adapter user guide

Reply:
Currently the information about the resource limit checks are not available in the user guide of the Kura adapter and the guide is not detailed as the other adapters . So I thought of a separate PR to update the user guide of the Kura adapter. Now I think, its better to update the user guide immediately. I have updated the user guide with the connection and connection duration limits information. Would you mind taking a look?

Reply:
It looks like you only have added a small paragraph but not the full information that is in the MQTT adapter. Is that by intention?

Reply:
yes. It is because that the user guide of Kura adapter is not well structured with sections as that of the MQTT adapter. For example, if I include the connection and duration limit sections (with heading) as in the MQTT adapter, then the following paragraph starting with "_Once the gateway has established a connection to the Kura adapter,_" doesn't have a heading and it doesn't looks nice. IMHO the paragraph that I added is not comprehensive, but it is _concise_ with needed information for a user.